Content.Server/Botany/Systems/SeedExtractorSystem.cs

```diff
@@ -40,15 +41,20 @@ private void OnInteractUsing(EntityUid uid, SeedExtractorComponent component, In
 
         QueueDel(args.Used);
 
-        var random = _random.Next(component.MinSeeds, component.MaxSeeds);
+        var amount = _random.Next(seedExtractor.BaseMinSeeds, seedExtractor.BaseMaxSeeds + 1) + seedExtractor.SeedAmountModifier;
         var coords = Transform(uid).Coordinates;
 
-        if (random > 1)
+        if (amount > 1)
             seed.Unique = false;
 
-        for (var i = 0; i < random; i++)
+        for (var i = 0; i < amount; i++)
         {
             _botanySystem.SpawnSeedPacket(seed, coords);
         }
     }
+
+    private void OnRefreshParts(EntityUid uid, SeedExtractorComponent seedExtractor, RefreshPartsEvent args)
+    {
+        seedExtractor.SeedAmountModifier = (int) args.PartRatings[seedExtractor.MachinePartSeedAmount] - 1;
```

Comment:
I made it percentage based now instead of increasing the min/max values. It defaults to 50% more seeds and can be easily tweaked later with `PartRatingSeedAmountMultiplier`.